src/front-end/typescript/lib/pages/opportunity/team-with-us/view.tsx

```diff
@@ -75,6 +82,7 @@ export type InnerMsg =
         api.ResponseValidation<TWUOpportunity, string[]>,
         TWUProposalSlim | null,
         api.ResponseValidation<Content, string[]>,
+        api.ResponseValidation<Content, string[]>,
```

Comment:
Is this change required here?  It looks like a duplication of line 84.

Reply:
yup, it's necessary. We're specifying what our messages should look like and those two happen to share a similar type (because they're both content).